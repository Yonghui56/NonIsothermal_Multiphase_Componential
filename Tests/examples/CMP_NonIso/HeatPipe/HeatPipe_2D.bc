GeoSys-BC: Boundary Conditions ------------------------------------------------
#BOUNDARY_CONDITION
 $PCS_TYPE
  CMP_NonIso_TotalDensityForm
 $PRIMARY_VARIABLE 
  MEAN_PRESSURE
 $GEO_TYPE
  POLYLINE PLY_LEFT
 $DIS_TYPE
  CONSTANT 101325

#BOUNDARY_CONDITION
 $PCS_TYPE
  CMP_NonIso_TotalDensityForm
 $PRIMARY_VARIABLE 
  TOTAL_MASS_DENSITY
 $GEO_TYPE
  POLYLINE PLY_LEFT
 $DIS_TYPE
  CONSTANT 7.19978e-3
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  CMP_NonIso_TotalDensityForm
 $PRIMARY_VARIABLE 
  TEMPERATURE
 $GEO_TYPE
  POLYLINE PLY_LEFT
 $DIS_TYPE
  CONSTANT 343.15
#STOP

