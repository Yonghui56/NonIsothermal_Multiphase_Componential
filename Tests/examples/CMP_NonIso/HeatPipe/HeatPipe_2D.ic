GeoSys-IC  Initial Conditions ------------------------------------------------
#INITIAL_CONDITION
 $PCS_TYPE
  CMP_NonIso_TotalDensityForm
 $PRIMARY_VARIABLE 
  MEAN_PRESSURE
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 101325

#INITIAL_CONDITION
 $PCS_TYPE
  CMP_NonIso_TotalDensityForm
 $PRIMARY_VARIABLE 
  TOTAL_MASS_DENSITY
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 7.19978e-3
  
#INITIAL_CONDITION
 $PCS_TYPE
  CMP_NonIso_TotalDensityForm
 $PRIMARY_VARIABLE 
  TEMPERATURE
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 343.15

#STOP


