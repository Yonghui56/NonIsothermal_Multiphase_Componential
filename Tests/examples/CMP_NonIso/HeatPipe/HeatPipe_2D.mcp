GeoSys-CP: Component Properties ------------------------------------------------

#COMPONENT_PROPERTIES  ; comp0
 $NAME
  Air ;
 $MOBILE 
  1; MOBIL-Flag: 0=immobile, 1=mobile/transported
 $DIFFUSION
  1  2.6e-6            	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp1
 $NAME
  Water ;
 $MOBILE
  1; MOBIL-Flag: 0=immobile, 1=mobile/transported
 $DIFFUSION
  1  0              ; diffusion model type, diffusion constant




#STOP