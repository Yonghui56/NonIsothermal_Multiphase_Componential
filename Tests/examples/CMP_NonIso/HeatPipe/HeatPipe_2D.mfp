GeoSys-MAT-FP: Fluid Properties ------------------------------------
#FLUID_PROPERTIES
 $FLUID_TYPE
  GAS
 $PCS_TYPE
  CMP_NonIso_TotalDensityForm
 $DENSITY
  1 0.9329
 $MOLAR_MASS
  1 0.02896
 $VISCOSITY
  1 1.67e-5
 $SPECIFIC_HEAT_CAPACITY
  1 1.180e+03
 $HEAT_CONDUCTIVITY
  1 0.582
#FLUID_PROPERTIES
 $FLUID_TYPE
  LIQUID
 $PCS_TYPE
  CMP_NonIso_TotalDensityForm
 $DENSITY
  1 1000.0
 $MOLAR_MASS
  1 0.01
 $VISCOSITY
  1 2.938e-4
 $SPECIFIC_HEAT_CAPACITY
  1 4.218e+03
 $HEAT_CONDUCTIVITY
  1 1.13
#STOP


#FLUID_PROPERTIES_NEW
$DENSITY
 0  1.000000e+003 ; density function, parameter
$VISCOSITY
 0  1.000000e-003 ; viscosity function, parameter
$HEAT_CAPACITY
 0  0.0
$HEAT_CONDUCTIVITY
 0 0.0 ; heat capacity, heat conductivity
$PHASE_DIFFUSION
 0  0.0

