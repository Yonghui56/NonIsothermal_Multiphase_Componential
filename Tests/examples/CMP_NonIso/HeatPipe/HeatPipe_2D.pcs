GeoSys-PCS: Processes ------------------------------------------------
#PROCESS
 $PCS_TYPE
 CMP_NonIso_TotalDensityForm
 $NUM_TYPE
   NEW   
#STOP

