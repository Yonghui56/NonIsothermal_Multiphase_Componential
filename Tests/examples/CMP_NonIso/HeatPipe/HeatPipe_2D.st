GeoSys-ST: Source Terms ------------------------------------------------
#SOURCE_TERM
 $PCS_TYPE
  CMP_NonIso_TotalDensityForm
 $PRIMARY_VARIABLE
  TEMPERATURE
 $GEO_TYPE
  POLYLINE PLY_RIGHT
 $DIS_TYPE   
  CONSTANT_NEUMANN 100 
#STOP


 $DIS_TYPE  
  CONSTANT_NEUMANN 0.0002

 $DIS_TYPE  
  RIVER 2
  4  3 1.00e-6 10.0 1.3 1.0
  5  3 1.00e-6 10.0 1.3 1.0