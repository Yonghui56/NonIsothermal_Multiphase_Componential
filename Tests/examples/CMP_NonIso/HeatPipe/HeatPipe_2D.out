GeoSys-OUT: Output ------------------------------------------------
#OUTPUT // profile
 $NOD_VALUES
  MEAN_PRESSURE
  TOTAL_MASS_DENSITY
  TEMPERATURE
 $ELE_VALUES
 Saturation
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
 STEPS 10
#STOP
STEPS 1


#OUTPUT // profile
 $NOD_VALUES
  Ca
  Cl
  K
  Na
  CaX2
  NaX
  KX
 $GEO_TYPE
  POINT  POINT2
 $DAT_TYPE
  PVD
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  Ca
  Cl
  K
  Na
  CaX2
  NaX
  KX
 $GEO_TYPE
  POLYLINE  OUT_LINE
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1


 $TIM_TYPE
   0.0
   180.0
   360.0
   8640.0
   86400.0

